defaults/main.yml

```diff
@@ -40,13 +40,13 @@ opn_check_mk_patches:
 # the old list syntax opn_check_mk_local_checks: [] is still supported
 opn_check_mk_local_checks:
   all:
-    - gateways_status.py
+    - check_powerunit.sh
```

Comment:
please be so kind and do not commit non-relevant changes; even if it is just a reordering of the existing list, it is irritating